mods/d2k/sequences/aircraft.yaml

```diff
@@ -2,7 +2,7 @@ carryall:
 	idle: DATA.R8
 		Start: 1923
 		Facings: -32
-		InterpolateIntermediateFacings: True
+		InterpolatedFacings: 1024
```

Comment:
Out of curiosity, will these interpolated facings be preloaded (added to the game's sprite sheets) or does the engine just rotate the 32 actual sprite frames on the fly, without any impact on the number of sheets/VRAM usage?

Reply:
Yes it rotates the 32 actual sprite frames on the fly to obtain the remaining 992 facings - no additional sprite sheets, facings or otherwise need to be loaded.